src/vs/workbench/api/browser/mainThreadSaveParticipant.ts

```diff
@@ -394,7 +392,6 @@ export class SaveParticipant implements ISaveParticipant {
 			model.textEditorModel.pushStackElement();
 		}, () => {
 			// user cancel
-			cts.dispose(true);
 		});
```

Comment:
No more user-cancelling. Better leave the `const cts = new Cancellation...` and pass the `token`-parameter as ctor-arg

Reply:
@jrieken got it, pushed